test/setup.js

```diff
@@ -0,0 +1,27 @@
+import { SpecReporter } from 'jasmine-spec-reporter';
+
+// in some tests, storybook takes a little while to start
+jasmine.DEFAULT_TIMEOUT_INTERVAL = 60000;
```

Comment:
Probably not necessary. Might still need to be 10s for Windows, but might not need to be this high (and we especially don't need the storybook comment)